scw/path.go

```diff
@@ -27,13 +27,27 @@ func inConfigFile() string {
 	return ""
 }
 
+// GetConfigPath returns the default path.
+// Default path is base on the following priority order:
+// - $SCW_CONFIG_PATH
+// - $XDG_CONFIG_HOME/scw/config.yaml
+// - $HOME/.config/scw/config.yaml
+// - $USERPROFILE/.config/scw/config.yaml
+func GetConfigPath() string {
+	configPath := os.Getenv(scwConfigPathEnv)
+	if configPath == "" {
+		configPath, _ = GetConfigV2FilePath()
+	}
+	return filepath.Clean(configPath)
+}
+
 // GetConfigV2FilePath returns the path to the Scaleway CLI config file
 func GetConfigV2FilePath() (string, bool) {
```

Comment:
I would rename function to `GetDefaultConfigPath`. This way `GetConfigPath` will try env and return `GetDefaultConfigPath()` by default

Reply:
We can also make this function private, it's confusing to have both `GetDefaultConfigPath` and `GetConfigPath`